packages/portal-proto/src/features/facets/NumericRangeFacet/types.ts

```diff
@@ -2,8 +2,8 @@ import { FacetCardProps, RangeFacetHooks } from "../types";
 
 export interface NumericFacetProps extends FacetCardProps<RangeFacetHooks> {
   readonly rangeDatatype: string;
-  readonly minimum?: number;
-  readonly maximum?: number;
+  readonly minimum: number;
```

Comment:
If these are required props, what is the case that the default value [here](https://github.com/NCI-GDC/gdc-frontend-framework/pull/1279/files#diff-b0e7154233efd3e562947e81932296bd4349b8971ad9818515bb22e9fd519cfbR40) gets used?

Reply:
Some of the values are "undefined" for which I am setting a default value.

Should this be?
```
  readonly minimum: number | undefined;
  readonly maximum: number | undefined;
```

Reply:
I would just leave them as optional props personally if they can be undefined but that works too.

Reply:
Why I am choosing the latter is because we want to pass in max and min values even though they are undefined. Previously, since it was optional we were not passing it leading to this bug.

Reply:
From a technical perspective there's no difference between passing in undefined and not passing in a prop. But if you want to convey that a value needs to be passed in, that works 👍